### DISCUSSION TITLE
Auth wallets and v2.0

### DISCUSSION THREAD

**Discussion by pyncz** [WalletConnect/web-examples#75] [Q&A]
Sorry if I am missing something, but I've just tried to catch https://react-auth-dapp.walletconnect.com and https://react-app.walletconnect.com working, but... Nothing happens after the scanning of the qr code.

I tried with:
- Omni wallet
- Zerion wallet

Although, they both are claimed to support v2.0 on the [explorer page](https://explorer.walletconnect.com/?type=wallet&version=2&page=1).

At the same time, demo wallets (https://react-auth-wallet.walletconnect.com and https://react-wallet.walletconnect.com, respectively) work fine ._.

What's up there? Are those issues on the wallets' side? How can we check what wallets _really_ implemented migration to v2.0 in this case?
Or were there some breaking changes since first stable versions of the clients?..

**Comment by chadyj** [comment]
Hey @pyncz.  Those wallets support Sign v2, but they don't yet support Auth since it is brand new.

We will update our explorer page to filter for SDK's.

**Reply by pyncz** [reply]
Hmm, yeah, that explains why I can't connect to the https://react-auth-dapp.vercel.com.

But what about the https://react-app.walletconnect.com? It uses not Auth but Sign client, so it should have been nice? And the [new web3-modal showcase](https://experiment-solib-integration.web3modal-dev.pages.dev/) doesn't work with those wallets neither. 🤔 

Seems like it's related to the whole v2 (its integration in the wallets, to be accurate)

**Reply by chadyj** [reply]
Fair comments!

I think you caught us in a transition moment because Auth was just released, and we haven't yet updated our examples. 

This is awesome feedback and I've shared with the team.

**Reply by pyncz** [reply]
:D
Ok, thank you :) It doesn't seem to be the examples' fault. So, I hope the proper integration by those and all the wallets will be done soon.

Will use demo wallets to play around v2 for now.

**Reply by pyncz** [reply]
@chadyj Sorry for being annoying. But!
I've checked the twitter accs of those wallets, and they all look like having migrated [eaaaarly](https://twitter.com/zerion_io/status/1560227526307299328?s=20&t=0zu1CTc87UBM2D9ESoyX2g), like in alfa / beta days.
My bet is that they simply haven't pick up [breaking changes with Pairing API](https://github.com/WalletConnect/walletconnect-monorepo/releases/tag/2.0.0), so their integrations just broke 😬

Maybe it's worth to re-validate wallets / dApps which support v2 and update the [explorer page](https://explorer.walletconnect.com/?type=wallet&version=2&page=1).

**Reply by chadyj** [reply]
No problem I'm happy to help clear things up.

V2 protocol has multiple SDK's.  See here https://docs.walletconnect.com/2.0/#getting-started

The early adopters are using Sign.  You can test this here https://react-app.walletconnect.com/
However, they have not yet adopted Auth.  

As mentioned above, we will update the explorer page to better reflect this.

**Reply by pyncz** [reply]
Nah... https://react-app.walletconnect.com/ also doesn't work with those wallets, that's the problem I tried to convey in the replies. So it's not just about Auth, it's about Sign client, as I [mentioned](https://github.com/WalletConnect/web-examples/discussions/75#discussioncomment-4079164).
